data/benefits/dynamic/fsl.ts

```diff
@@ -7,527 +7,385 @@ const DEFAULT_FSL = {
   },
-  D95: {
+  departement_val_d_oise: {
     label: "du département du Val d’Oise",
-    resources: {
-      link: "https://www.valdoise.fr/aide-et-service/11/6-fonds-de-solidarite-logement-aide-a-l-acces-au-logement.htm",
-    },
+    link: "https://www.valdoise.fr/aide-et-service/11/6-fonds-de-solidarite-logement-aide-a-l-acces-au-logement.htm",
   },
-  D974: {
+  departement_la_reunion: {
     label: "du département de la Réunion",
-    resources: {
-      link: "https://www.departement974.fr/aide/aide-habitat-fonds-de-solidarite-pour-logement-fsl#aidesimpayes",
-    },
+    link: "https://www.departement974.fr/aide/aide-habitat-fonds-de-solidarite-pour-logement-fsl#aidesimpayes",
   },
 }
 
-function getDepartmentInstitutionByInseeCode(institutionsMap, inseeCode) {
-  return Object.keys(institutionsMap)
-    .filter(
-      (institutionName) =>
-        institutionsMap[institutionName].type === "departement"
-    )
-    .find((institutionName) => {
-      const institution = institutionsMap[institutionName]
-      return institution.code_insee === inseeCode
-    })
-}
-
-function getMetropoleInstitutionBySirenCode(institutionsMap, sirenCode) {
-  return Object.keys(institutionsMap).find((institutionName) => {
-    const institution = institutionsMap[institutionName]
-    return institution.code_siren === sirenCode
-  })
-}
-
-function formatBenefit(customizationBenefit, institution) {
+function formatBenefit(
+  { label, link, form, instructions, excludedEPCI },
+  institutionId
+) {
   return {
-    id: `${institution.replace(/_/g, "-")}-fsl-eligibilite`,
+    id: `${institutionId.replace(/_/g, "-")}-fsl-eligibilite`,
     ...DEFAULT_FSL,
-    label: `Aide au maintien dans votre logement ${customizationBenefit.label}`,
-    description: `Dans le cadre du Fonds de Solidarité Logement ${customizationBenefit.label}, des aides financières sont mises en place pour vous aider à rester dans votre logement et à payer vos factures liées à votre logement (eau, électricité, etc.).`,
+    description: `Dans le cadre du Fonds de Solidarité Logement ${label}, des aides financières sont mises en place pour vous aider à rester dans votre logement et à payer vos factures liées à votre logement (eau, électricité, etc.).`,
     conditions: [
-      `Occuper, à titre de résidence principale, un logement sur le territoire ${customizationBenefit.label}.`,
+      `Occuper, à titre de résidence principale, un logement sur le territoire ${label}.`,
       "<strong>Satisfaire les conditions de ressources</strong> décrites dans le règlement.",
     ],
-    ...customizationBenefit.resources,
-    institution,
+    link,
+    form,
+    instructions,
+    label: `Aide au maintien dans votre logement ${label}`,
+    institution: institutionId,
     source: "javascript",
     conditions_generales: [
       {
         type: "attached_to_institution",
       },
+      ...(excludedEPCI
+        ? [
+            {
+              type: "invert",
```

Comment:
On en avait déjà parlé mais je trouverais un `not` ou un `exclude` plus clair qu'un invert